src/main/java/org/zalando/pazuzu/feature/FeatureService.java

```diff
@@ -40,7 +45,8 @@ public FeatureService(FeatureRepository featureRepository) {
     }
 
     @Transactional(rollbackFor = ServiceException.class)
-    public <T> T createFeature(String name, String dockerData, String testInstruction, String description, List<String> dependencyNames, Function<Feature, T> converter) throws ServiceException {
+    public <T> T createFeature(String name, String dockerData, String testInstruction, String description,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method createFeature() has an NPath complexity of 360](https://www.codacy.com/app/Pazuzu/pazuzu-registry/file/2764791632/issues/source?bid=3131965&fileBranchId=3355916#l48)